ui/src/app/pages/data/data-set/export-data-set-modal.tsx

```diff
@@ -82,15 +82,22 @@ class ExportDataSetModal extends React.Component<
     }
   }
 
-  async generateQuery() {
-    const {dataSet, workspaceNamespace, workspaceFirecloudName} = this.props;
-    const dataSetRequest: DataSetRequest = {
+  get datasetRequest() {
+    const {dataSet} = this.props;
+    return {
       name: dataSet.name,
+      includesAllParticipants: dataSet.includesAllParticipants,
+      description: dataSet.description,
       conceptSetIds: dataSet.conceptSets.map(cs => cs.id),
       cohortIds: dataSet.cohorts.map(c => c.id),
       domainValuePairs: dataSet.domainValuePairs,
-      includesAllParticipants: dataSet.includesAllParticipants,
+      prePackagedConceptSet: dataSet.prePackagedConceptSet
     };
```

Comment:
This change removes type info - we should keep that, maybe with `as DataSetRequest` ?

Good to see this consolidation of code.